packages/api/src/record/validators.ts

```diff
@@ -42,3 +42,28 @@ export function validatePatchRecordRequest(
     throw validation.error;
   }
 }
+
+export function validateCopyRecordRequest(
+  data: unknown
+): asserts data is CopyRecordRequest {
+  const validation = Joi.object()
+    .keys({
+      ...fieldsFromUserAuthentication,
+      RecordVO: {
+        recordId: Joi.number().integer().required(),
+        archiveNbr: Joi.string().optional().allow(null),
+        folder_linkId: Joi.number().integer().optional().allow(null),
+        parentFolder_linkId: Joi.number().integer().optional().allow(null),
+        parentFolderId: Joi.number().integer().optional().allow(null),
+        folderId: Joi.number().integer().optional().allow(null),
+        uploadFileName: Joi.string().optional().allow(null),
+      },
+      FolderDestVO: { folder_linkId: Joi.number().integer().required() },
```

Comment:
I'd use `folderId` rather than `folder_linkId`. Ideally callers of the API never need know what a `folder_link` is.